### DISCUSSION TITLE
URL on installation instructions points to a resource that does not exist

### DISCUSSION THREAD

**Discussion by MakonnenMak** [com-lihaoyi/mill#2190] [Q&A] [answered]
Hey! Trying to install Mill with instructions on https://com-lihaoyi.github.io/mill/mill/Installation.html 

The url we fetch the Mill package from seems to point to a non-existent resource

`curl -L https://github.com/com-lihaoyi/mill/releases/download/0.11.0-M0/0.11.0-M0 > mill && chmod +x mill`

I get a 404 response. What should tihs poit to instead ?

**Comment by hmf** [comment]
Take a look at the releases page. You will find:

https://github.com/com-lihaoyi/mill/releases/tag/0.11.0-M0

HTHs

**Comment by lefou** [comment, marked as answer]
Thank for reporting this. A fix is in the pipeline. 

* #2192 

The latest stable Mill version is to date `0.10.10` and the link in the [documentation of that version](https://com-lihaoyi.github.io/mill/mill/0.10.10/Installation.html) is correct. 

* mill bootstrap script: https://github.com/com-lihaoyi/mill/releases/download/0.10.10/0.10.10

**Comment by MakonnenMak** [comment]
@lefou  thanks, yep using  latest stable version works just fine
